9.4.методы_строк_часть2/9.4.12.py

```diff
@@ -0,0 +1,5 @@
+site = str(input())
+if site.endswith(".com") or site.endswith(".ru"):
```

Comment:
Метод `.endswith()` позволяет проверить несколько значений передав в аргумент не строку, а кортеж строк:
```suggestion
if site.endswith((".com", ".ru")):
```